runtime/doc/index.txt

```diff
@@ -1441,6 +1441,9 @@ tag		command		action ~
 |:packloadall|	:packl[oadall]	load all packages under 'packpath'
 |:pclose|	:pc[lose]	close preview window
 |:pedit|	:ped[it]	edit file in the preview window
+|:perl|		:perl		execute perl command
```

Comment:
Extra tab?

Reply:
Thought so as well, but it's just the way I think github renders it.